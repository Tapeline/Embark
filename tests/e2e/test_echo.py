from tests.runner import run_embark


def test_echo_task(snapshot):
    """Test that std.echo works."""
    proc = run_embark("tests/fixtures/test_echo.yml")
    assert proc.returncode == 0, (proc.stdout, proc.stderr)
    assert proc.stdout == snapshot
