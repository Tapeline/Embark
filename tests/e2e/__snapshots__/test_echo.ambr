# serializer version: 1
# name: test_echo_task
  '''
  Hello, World!
  
  '''
# ---
